### PR TITLE
Add Log4j2 mitigation steps to ES tuning section

### LINKED ISSUE #4679: Add log4j mitigation to Elasticsearch tuning
Hello team!

I have answered a community user and I have seen that mitigation of the Log4j vulnerability has been added in the Elasticsearch installation and upgrade guides. It should be added too to the `User Manual - Elasticsearch tuning` section, since there may be users who do not apply this fix during the installation or already have Elasticsearch installed from before, also, it would be interesting to reflect the blog that was just published about the vulnerability: https://wazuh.com/blog/detecting-log4shell-with-wazuh/

Regards, Raúl.

### PR DESCRIPTION
**Pull request by javimed**
## Description

This PR closes #4679. It adds instructions on how to mitigate Log4Shell vulnerability and it also fixes References titles.

## Checks
- [X] It compiles without warnings.
- [X] Spelling and grammar. 
- [ ] Used impersonal speech. 
- [X] Used uppercase only on nouns. 
- [X] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/user-manual/elasticsearch/elastic-tuning.rst [hunk 1]
```diff
@@ -457,6 +459,48 @@ The number of replicas can be changed dynamically using the Elasticsearch API. I
...
+        # echo '-Dlog4j2.formatMsgNoLookups=true' > /etc/elasticsearch/jvm.options.d/disabledlog4j.options
+
+  #. Set file's users and permissions.
+    
+      .. code-block:: console
+
+        # chmod 2750 /etc/elasticsearch/jvm.options.d/disabledlog4j.options
+        # chown root:elasticsearch /etc/elasticsearch/jvm.options.d/disabledlog4j.options
+
+  #. Restart the Elasticsearch service
```
Comment (s-ocando) [inline source/user-manual/elasticsearch/elastic-tuning.rst:496]:
```suggestion
  #. Restart the Elasticsearch service.
```

File: source/user-manual/elasticsearch/elastic-tuning.rst [hunk 2]
```diff
@@ -457,6 +459,48 @@ The number of replicas can be changed dynamically using the Elasticsearch API. I
...
+
+
+.. _mitigate_Log4j2:
+
+Mitigate Log4j2 vulnerability
+-----------------------------
+
+Although Wazuh is not affected by the Apache Log4j2 Remote Code Execution vulnerability (CVE-2021-44228), we still recommend to apply a workaround.
+
+To mitigate the Log4Shell vulnerability, take the following steps on each one of the Elasticsearch/ODfE nodes.
```
Comment (s-ocando) [inline source/user-manual/elasticsearch/elastic-tuning.rst:475]:
```suggestion
To mitigate the Log4Shell vulnerability, take the following steps on each one of the Elasticsearch nodes.
```

**Review by MarVergara — APPROVED** [review]
LGTM - editorial review.

**Comment by MarVergara** [comment]
I am closing this PR since the approach to this fix has changed.

**Pull request closed without merging**